tests/lib/state_test.py

```diff
@@ -277,11 +280,16 @@ def _base_state(self):
             {
                 Interface.KEY: [
                     {
-                        "name": "foo-name",
-                        "type": "foo-type",
-                        "state": "up",
-                        "bridge": {
-                            "port": [{"name": "eth0", "type": "system"}]
+                        Interface.NAME: "foo-name",
+                        Interface.TYPE: "foo-type",
+                        Interface.STATE: InterfaceState.UP,
+                        OVSBridge.CONFIG_SUBTREE: {
+                            OVSBridge.PORT_SUBTREE: [
+                                {
+                                    OVSBridge.Port.NAME: "eth0",
+                                    Interface.TYPE: "system",
```

Comment:
Please, remove this. `type` is not needed anymore in `OVSBridge` port subtree. Thanks!

Reply:
Fixed it, thanks!